app/scripts/lib/rpc-method-middleware/handlers/index.ts

```diff
@@ -35,4 +34,4 @@ const handlers = [
   ///: END:ONLY_INCLUDE_IF
 ];
 
-export default handlers;
+export const legacyHandlers = [ethAccounts];
```

Comment:
not a huge deal but wondering if "legacy" is the right framing here? It seems like the main reason this is separated out is that you need to be able to call this method and receive an empty array if there is no authorization? Perhaps that is a legacy pattern in that we don't want to it to be reused but legacyHandlers implies to me that this is method that is set for deprecation or something like that

Reply:
> Perhaps that is a legacy pattern in that we don't want to it to be reused

maybe this is the proper use of legacy... 🤔